src/collections/Events/index.ts

```diff
@@ -0,0 +1,71 @@
+import type { CollectionConfig } from 'payload'
+
+import { authenticated } from '../../access/authenticated'
+import { authenticatedOrPublished } from '../../access/authenticatedOrPublished'
+
+export const Events: CollectionConfig = {
+  slug: 'events',
+  access: {
+    create: authenticated,
+    delete: authenticated,
+    read: authenticatedOrPublished,
+    update: authenticated,
+  },
+  admin: {
+    useAsTitle: 'title',
+  },
+  fields: [
+    {
+      name: 'title',
+      type: 'text',
+      required: true,
+    },
+    {
+      name: 'date',
+      type: 'date',
+      required: true,
+    },
+    {
+      name: 'end date',
```

Comment:
I think this (and the other names) are better off in camel case (eg; endDate) since this allows us to use the .endDate syntax (rather than being restricted to using `events["end date"]`)